### PR TITLE
Add info on where data is stored

### PR DESCRIPTION
**Pull request by jackbridger**
## Summary

This is a docs change and relates to issue #905  - adding information on where data is stored.

I placed it under the advanced category in the sidebar. Let me know if you prefer it somewhere else. 

CleanShot 2023-02-07 at 10 49 59

## Test Plan

It is a docs change so likely just need to review that the docs run as they should. Let me know if you want me to do any other testing.

Thanks!!

### CODE REVIEW

File: website/docs/advanced/Where-data-stored.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+id: where_data_stored
+title: Where your data is stored
+sidebar_label: Where data is stored
+---
+
+* <b>Android</b> - SQLite
```
Comment (tido64) [inline website/docs/advanced/Where-data-stored.md:7]: Can we replace `<b>` and `</b>` with `**` instead?
```suggestion
* **Android** - SQLite
```

File: website/docs/advanced/Where-data-stored.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+* <b>iOS</b> - small values (not exceeding 1024 characters) are serialized and stored in a common `manifest.json` file, while larger values are stored in individual, dedicated files (named as MD5 hashed `key`)
+* <b>Web</b> - window.localStorage
+* <b>MacOS</b> - Same as iOS
```
Comment (tido64) [inline website/docs/advanced/Where-data-stored.md:10]: _macOS_ is stylized with a lower case "m"

```suggestion
* **macOS** - Same as iOS
```

File: website/docs/advanced/Where-data-stored.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+* <b>Windows</b> - SQLite
```
Comment (tido64) [inline website/docs/advanced/Where-data-stored.md:11]: Nit: It would also be nice if we could sort these alphabetically.

**Review by tido64 — COMMENTED** [review]
Thanks for adding this 👍

File: website/docs/advanced/Where-data-stored.md (hunk 2 above)
Comment (jackbridger) [reply website/docs/advanced/Where-data-stored.md:10]: good point!

File: website/docs/advanced/Where-data-stored.md (hunk 3 above)
Comment (jackbridger) [reply website/docs/advanced/Where-data-stored.md:11]: Updating now!

**Comment by jackbridger** [comment]
Everything should be updated now
CleanShot 2023-02-07 at 11 21 29

**Review by tido64 — APPROVED** [review]
Thanks 🙏

**Comment by krizzu** [comment]
Thanks Jack 🙏 
@tido64 seems like Windows cannot resolve some dependencies, again. I think we can just move this forward anyway.

**Review by krizzu — APPROVED** [review]

**Comment by jackbridger** [comment]
Yay! Thanks both for your support. This was my first contribution to a project that lots of people use. Even though it's only docs, it feels good. Hope it's the start of more contributions. Thanks!!

**Pull request merged**